setup.py

```diff
@@ -69,6 +69,7 @@ def find_version(*file_paths):
             'sphinx-gallery',
             'sphinx_rtd_theme',
             'nbsphinx',
+            'flaky',
```

Comment:
flaky doesn't seem used outside of test.

Reply:
Yes, it's declared here for 'dev' purpose

Reply:
My expectation is that 'pip install .[dev]' will install all development (and testing) dependencies